parsing/parchmint-parser.js

```diff
@@ -628,6 +626,28 @@ class ParchmintParser {
         return true;
     }
 
+    /**
+     * Clear all data from the parser.
+     *
+     * Reset the parser to its beginning state. This method does not reallocate
+     * anything except the layers array.
+     *
+     * @since 1.0.0
+     */
+    clear() {
+        this.parchmint = Validation.DEFAULT_STR_VALUE;
+
+        this.valid = true;
+        this.idSet.clear();
+
+        this.architecture = null;
+        this.layers = [];
+        this.components.clear();
+        this.connections.clear();
+        this.compFeatures.clear();
+        this.connFeatures.clear();
+    }
```

Comment:
I believe this was also removed in the last PR, bad merge?

Reply:
Thank you for catching that I totally missed it. It's been removed.